src/main/java/nl/tudelft/lifetiles/graph/models/jgrapht/JGraphTGraphAdapter.java

```diff
@@ -182,4 +190,17 @@ private DefaultEdge unpackEdge(final Edge<V> input) {
         return ed.getInternalEdge();
     }
 
+    /**
+     * @author Rutger van den Berg
+     *         Compares two edges by their target vertex.
+     */
+    class EdgeComparatorByVertex implements Comparator<Edge<V>> {
```

Comment:
This class is redundant, you could rewrite these 10 lines to a one-liner by using a lambda. See http://www.dreamsyssoft.com/java-8-lambda-tutorial/comparator-tutorial.php for axamples.